src/Commands/MigrateFreshCommand.php

```diff
@@ -1,91 +0,0 @@
-<?php
-
-namespace Nwidart\Modules\Commands;
-
-use Illuminate\Console\Command;
-use Nwidart\Modules\Traits\ModuleCommandTrait;
-use Symfony\Component\Console\Input\InputArgument;
-use Symfony\Component\Console\Input\InputOption;
-
-class MigrateFreshCommand extends Command
-{
-    use ModuleCommandTrait;
-
-    /**
-     * The console command name.
-     *
-     * @var string
-     */
-    protected $name = 'module:migrate-fresh';
-
-    /**
-     * The console command description.
-     *
-     * @var string
-     */
-    protected $description = 'Drop all database tables and re-run all migrations';
-
-    /**
-     * Execute the console command.
-     */
-    public function handle(): int
-    {
-        $module = $this->argument('module');
-
-        if ($module && !$this->getModuleName()) {
-            $this->error("Module [$module] does not exists.");
-
-            return E_ERROR;
-        }
-
-        $this->call('migrate:fresh');
```

Comment:
This command should be enhanced to affect only the specified module. Instead of dropping the feature entirely!

For instance, if a developer executes the `php artisan module:migrate-fresh User`, it should refresh migrations from the User module exclusively. If no module is provided, the command should execute for all modules.